src/components/organisms/MapboxMap/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import ReactMapGL, { NavigationControl } from 'react-map-gl'
+
+class Mapbox extends Component {
+  state = {
+    viewport: {
+      width: 400,
```

Comment:
it should be a prop, doesn't?

Reply:
I could be wrong, but I think it need be a prop because onviewportChange it'll call the updateViewport function that updates the state

Reply:
I'll test it